libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/PrettyCore.hs

```diff
@@ -1273,7 +1272,7 @@ ppNEWEPOCH Allegra x = ppShelleyNewEpochPredicateFailure x
 ppNEWEPOCH Mary x = ppShelleyNewEpochPredicateFailure x
 ppNEWEPOCH Alonzo x = ppShelleyNewEpochPredicateFailure x
 ppNEWEPOCH Babbage x = ppShelleyNewEpochPredicateFailure x
-ppNEWEPOCH Conway x = ppConwayNewEpochPredFailure x
+ppNEWEPOCH Conway _ = ppString "PredicateFailure (ConwayNEWEPOCH era) = Void, and can never Fail"
```

Comment:
```suggestion
ppNEWEPOCH Conway x = absurd x
```

Reply:
There is no need for this, since this pretty core module is being obliterated as we speak: #4966